components/bl-pdfViewer-component/src/index.js

```diff
@@ -27,17 +30,24 @@ export default function PdfViewer({ component, eventHandlers, elRef }) {
     }
   }, [documentRef, height, width]);
 
+  useEffect(() => {
+    if (pageLoaded) {
+      setRootHeight(elRef.current.getBoundingClientRect().height);
+      setRootWidth(elRef.current.getBoundingClientRect().width);
+    }
+  }, [pageLoaded, elRef, height, width]);
+
   useEffect(() => {
     if (pageLoaded) {
       const spaceForControls = getBottomOffset(controlsRef.current) - getBottomOffset(pageRef);
 
-      documentRef.style.height = `calc(${ height } - ${ spaceForControls }px)`;
-      documentRef.style.width = width || 'auto';
+      documentRef.style.height = rootHeight - spaceForControls + 'px';
```

Comment:
We could make it more obvious

`${rootHeight - spaceForControls}px`

same below

Reply:
Also, it could be an utility function like

`usePx` `addPx` or simply `px(variable)`, measure(variable, 'px') choose which you like